src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputMethodParamPad.java

```diff
@@ -79,5 +79,5 @@ public void newArray()
             ()
         {
         }
-    }
+    }	
```

Comment:
you need to practicing to deliver only requested changes. You need to review your changes/PR yourself to verify that all is fine.
Messing all together is bad practice in most teams.